README.md

```diff
@@ -155,6 +155,9 @@ artifactory-cleanup --output=myfile.txt
 
 # Save the summary in a Json file
 artifactory-cleanup --output=myfile.txt --output-format=json
+
+# Save the summary in a json file and append a list with removed artifacts
+artifactory-cleanup --output=myfile.json --output-format json-with-artifact-list
```

Comment:
We shouldn't introduce complete new "format" for that.

That's how we could add it:
```
artifactory-cleanup --output=myfile.txt --output-format=json --output-artifacts
```

So we can extend this approach later
```
artifactory-cleanup --output=myfile.txt --output-format=json --output-artifacts --output-skipped-artifacts --output-whateveryou-want-data
```

Reply:
When I tested it, I wondered why it didn't work with --output-artifacts. The reason, of course, is that it only works with JSON.

If, for example, markdown is added as an output format, the option will only work with one of three formats. If Markdown or Table were to get another output format, it would become very confusing. That's why I prefer extending --output-format. This way, it becomes just a list.

Reply:
It would actually be nice to have subcommands for the  formats, allowing us to attach the relevant options directly to them.